go/kbfs/tlfhandle/resolve.go

```diff
@@ -834,11 +835,16 @@ func parseHandleLoose(
 			if iteamHandle.tlfID != tlf.NullID {
 				return iteamHandle, nil
 			}
+		} else {
+			// This is not an implicit team, so continue on to check for a
+			// normal team.  TODO: return non-nil errors immediately if they
+			// don't simply indicate the implicit team doesn't exist yet
+			// (i.e., when we start creating them by default).
+			switch errors.Cause(err).(type) {
```

Comment:
You'll probably get a lint error about a switch with only one case in it.  I think it would be clearer (and the comment would make more sense) as something like:

```go
} else {
			switch errors.Cause(err).(type) {
			case libkb.TeamContactSettingsBlockError:
				// The implicit team couldn't be created due to one of the
				// users' privacy settings, so fail the handle lookup completely.
				return nil, err
			default:
				// This is not an implicit team, so continue on to check for a
				// normal team.  TODO: return non-nil errors immediately if they
				// don't simply indicate the implicit team doesn't exist yet
				// (i.e., when we start creating them by default).
			}
```